apps/passport-client/src/useSyncE2EEStorage.tsx

```diff
@@ -203,6 +221,13 @@ export async function downloadStorage(
       storageResult.value.storage,
       await getPackages()
     );
+
+    const validationErrors = validatePCDCollection(pcds);
+    if (validationErrors.length > 0) {
+      logAndUploadValidationErrors(validationErrors);
+      throw new Error("validation errors:\n" + validationErrors.join("\n"));
```

Comment:
This function should be returning a result with success: false rather than throwing.  It might be worth trying to inform the user that we detected a bad server-side state, but in this case we don't want to lock them out if the local state is good.  In theory, the fix to that would be to force an upload of the local state somehow.